examples/hybrid/callbacks.jl

```diff
@@ -203,8 +203,8 @@ function save_to_disk_func(integrator)
             )
 
         # rain vs snow
-        ᶜ3d_rain = @. ifelse(ᶜT >= FT(273.15), ᶜS_ρq_tot, FT(0))
-        ᶜ3d_snow = @. ifelse(ᶜT < FT(273.15), ᶜS_ρq_tot, FT(0))
+        @. ᶜ3d_rain = ifelse(ᶜT >= FT(273.15), ᶜS_ρq_tot, FT(0))
```

Comment:
`ᶜ3d_rain` needs to be unpacked from `p` above

Reply:
gotcha

Reply:
Seems like we never use it in our CI. I'll switch it on for one case just to test it